app/src/main/java/com/kaamchor/android/fameofshame/CountDownActivity.java

```diff
@@ -0,0 +1,40 @@
+package com.kaamchor.android.fameofshame;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.view.Window;
+import android.widget.NumberPicker;
+
+public class CountDownActivity extends AppCompatActivity {
+
+    int hr = 0;
+    int min = 0;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_count_down);
+        NumberPicker hour = (NumberPicker) findViewById(R.id.hour);
```

Comment:
you can probably make the UI objects global, and get a reference to them in onCreate like you have done here.
This way if you have to use the ui element at some other place, you wont have to keep getting a reference to it everytime